src/main/java/core/basesyntax/exception/UserAgeException.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.exception;
+
+public class UserAgeException extends RuntimeException {
```

Comment:
```suggestion
public class RegistrationException extends RuntimeException {
```

Remove all other exceptions